sentry-debug-images/src/unix.rs

```diff
@@ -1,10 +1,38 @@
 use std::env;
 
-use findshlibs::{Segment, SharedLibrary, SharedLibraryId, TargetSharedLibrary, TARGET_SUPPORTED};
 use sentry_core::protocol::debugid::DebugId;
 use sentry_core::protocol::{DebugImage, SymbolicDebugImage};
 use sentry_core::types::Uuid;
 
+use findshlibs::{Segment, SharedLibrary, SharedLibraryId, TargetSharedLibrary, TARGET_SUPPORTED};
+
+const UUID_SIZE: usize = 16;
+
+/// Converts an ELF object identifier into a `DebugId`.
+///
+/// The identifier data is first truncated or extended to match 16 byte size of
+/// Uuids. If the data is declared in little endian, the first three Uuid fields
+/// are flipped to match the big endian expected by the breakpad processor.
+///
+/// The `DebugId::appendix` field is always `0` for ELF.
+fn debug_id_from_build_id(build_id: &[u8]) -> Option<DebugId> {
+    let mut data = [0 as u8; UUID_SIZE];
```

Comment:
Ideally, we could use `uuid::Bytes` here, and get rid of the constant, but that's not exposed from `sentry-types`.